src/components/form-field/form-field/form-field.ts

```diff
@@ -154,7 +154,7 @@ export class SbbFormField extends LitElement {
   }
 
   protected override willUpdate(changedProperties: PropertyValues<this>): void {
-    if (changedProperties.has('label')) {
+    if (changedProperties.has('label') && isBrowser()) {
```

Comment:
Maybe it would be more safe to move the check into _renderLabel, in case renderlabel would be used at another place?